### PR TITLE
HBASE-24989 [HBOSS] Some code cleanup

### PR DESCRIPTION
**Pull request by wchevreuil**

### CODE REVIEW

File: hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/TreeLockManager.java [hunk 1]
```diff
@@ -231,25 +231,27 @@ private boolean retryBackoff(int retries) throws IOException {
    * or write locks should be held by anyone on any child directory.
    *
    * @param path Path to lock
+   * @param depth Depth.DIRECTORY to look for locks on this path only, or Depth.RECURSIVE to
```
Comment (HorizonNet) [inline hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/TreeLockManager.java:234]: Would it be better to use a Javadoc link for the `Depth` parts?

**Review by HorizonNet — COMMENTED** [review]
Left a question for now.

File: hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/TreeLockManager.java (hunk 1 above)
Comment (wchevreuil) [reply hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/TreeLockManager.java:234]: Thanks for reviewing it. Yes, and while checking on this, I realised there are 25 more javadoc warnings. Will work on all of them and push in the next commit.

**Review by joshelser — APPROVED** [review]
LGTM, nice cleanup

**Review by HorizonNet — APPROVED** [review]
LGTM

**Pull request merged**